index.js

```diff
@@ -37,20 +38,18 @@ var sourcemapReporter = function(file) {
       colNum = message.col;
     }
 
-    gutil.log(
-      c.red('[') +
-      (
-        typeof message.line !== 'undefined' ?
-          c.yellow( 'L' + lineNum ) +
-          c.red(':') +
-          c.yellow( 'C' + colNum )
-        :
-          c.yellow('GENERAL')
-      ) +
-      c.red('] ') +
-      message.message + ' ' + message.rule.desc + ' (' + message.rule.id + ')');
+    var msgInfo = message.message + ' ' + message.rule.desc + ' (' + message.rule.id + ')';
+
+    var locInfo = c.red('(') + c.yellow(path.relative(process.cwd(), originalPos.source) + ':');
+    if (typeof message.line !== 'undefined') {
+      locInfo += c.yellow(lineNum) + c.red(':') + c.yellow(colNum);
+    } else {
+      locInfo += c.yellow('GENERAL');
+    }
+    locInfo += c.red(')');
+
+    console.log(msgInfo, locInfo, '\n');
```

Comment:
> I've used console.log because the printed timestamp (with gutil.log) is too verbose for each error of csslint.

I agree.

> For style issues, line break, etc. please add a coding style guideline jscsrc.json. I've autoformated the code with my default coding style config.

I meant that  `console.log(msgInfo, locInfo, '\n');` should be `console.log(msgInfo, locInfo);`.
But now, I understand why you add the `\n`, because CSSLint messages are too long.